.buildkite/agents.json

```diff
@@ -0,0 +1,79 @@
+{
+  "gcp": {
+    "project": "elastic-kibana-ci",
+    "zones": ["us-central1-a", "us-central1-b", "us-central1-c", "us-central1-f"],
+    "serviceAccount": "elastic-buildkite-agent@elastic-kibana-ci.iam.gserviceaccount.com",
+    "imageFamily": "kb-ubuntu",
+    "subnetwork": "buildkite",
+    "disableExternalIp": true,
+    "diskType": "pd-ssd",
+    "diskSizeGb": 75,
+    "overprovision": 0,
+    "minimumAgents": 0,
+    "maximumAgents": 50,
+    "gracefulStopAfterMins": 360,
+    "hardStopAfterMins": 540,
+    "idleTimeoutMins": 10,
+    "exitAfterOneJob": false,
+
+    "agents": [
+      {
+        "queue": "default",
+        "name": "kb-default",
+        "minimumAgents": 1,
+        "maximumAgents": 100,
```

Comment:
Do we want a max here?

Reply:
Obviously not a blocker to this PR since we're just talking baseline jobs.

Reply:
I like putting limits in just so that something doesn't accidentally spin up agents out of control somehow, and they are easy to adjust. These `default` agents typically only run really short/quick steps, so I'm guessing the number of these would actually stay under 10 or so most of the time